<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formoza</string>
  </property>
  <widget class="QWidget" name="Zakladki">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="Ogloszenia">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Ogłoszenia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <widget class="QScrollArea" name="Ogloszenia_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="Ogloszenia_scrollArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>70</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QListView" name="Ogloszenia_list"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Ogloszenia_temat">
          <property name="text">
           <string>Temat</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="Ogloszenia_text"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Wydarzenia">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Wydarzenia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="3">
         <widget class="QLabel" name="Wydarzenia_temat">
          <property name="text">
           <string>Temat</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="3">
         <widget class="QLabel" name="Wydarzenia_info">
          <property name="text">
           <string>Dodatkowe Info</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="Wydarzenia_decyzja">
          <property name="text">
           <string>Weźmiesz udział?</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QPushButton" name="Wydarzenia_b_tak">
          <property name="text">
           <string>Tak</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="Wydarzenia_b_nie">
          <property name="text">
           <string>Nie</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4" colspan="3">
         <widget class="QScrollArea" name="Wydarzenia_scroll">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="Wydarzenia_scrollArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>70</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QListWidget" name="Wydarzenia_list">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="batchSize">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QTextBrowser" name="Wydarzenia_text"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Grupy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Grupy</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" rowspan="2">
         <widget class="QScrollArea" name="Grupy_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="Grupy_scrollArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>70</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QListWidget" name="Grupy_list"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Grupy_opis">
          <property name="text">
           <string>Tytuł</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTabWidget" name="Grupy_tab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Grupy_Czlonkowie">
           <attribute name="title">
            <string>Członkowie</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QScrollArea" name="Grupy_Czlonkowie_scroll">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="Grupy_Czlonkowie_scrollArea">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>98</width>
                 <height>70</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_11">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QListWidget" name="Grupy_Czlonkowie_list"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Grupy_Ogloszenia">
           <attribute name="title">
            <string>Ogłoszenia</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="Grupy_Dyskusje">
           <attribute name="title">
            <string>Dyskusje</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="OMnie">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>O Mnie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="OMnie_imie">
          <property name="text">
           <string>Imię</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="OMnie_nazwisko">
          <property name="text">
           <string>Nazwisko</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="OMnie_stopien">
          <property name="text">
           <string>Stopień</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="OMnie_funkcja">
          <property name="text">
           <string>Funkcja</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Ustawienia">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Ustawienia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="Ustawienia_login_input"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Ustawienia_login">
          <property name="text">
           <string>Login:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Ustawienia_haslo">
          <property name="text">
           <string>Hasło:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="Ustawienia_haslo_input">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="Ustawienia_b_zaloguj">
          <property name="text">
           <string>Zaloguj</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="Ustawienia_status">
          <property name="text">
           <string>Status zalogowania</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="Ustawienia_wersja">
          <property name="text">
           <string>Wersja programu</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Ustawienia_b_aktualizuj">
          <property name="text">
           <string>Aktualizuj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
